cabal-testsuite/PackageTests/ProjectConfig/FieldStanzaConfusion/cabal.out

```diff
@@ -1,4 +1,13 @@
 # cabal build
-Error: [Cabal-7090]
-Error parsing project file cabal.project:4:
-  - 'source-repository-package' is a stanza, not a field. Remove the trailing ':' to parse a stanza.
+Error: [Cabal-7167]
+Error encountered when parsing project file cabal.project:
+
+cabal.project:4:1: error:
+'source-repository-package' is a stanza, not a field. Remove the trailing ':' to parse a stanza.
+
+    1 | packages: .
+    2 | 
+    3 | -- This is an error; a trailing `:` is syntax for a field, not a stanza!
+    4 | source-repository-package:
+      | ^
```

Comment:
Do we need all of lines 1..4? Could we number only the problem line and include the lines above and below, unnumbered. This is what GHC does, isn't it?

Reply:
Again, better in a separate MR I think, as it's how cabal file parse errors are already displayed.

Reply:
I agree and have raised issue #11091.